### PR TITLE
CI fixes

### PR DESCRIPTION
**Pull request by rorp**

### CODE REVIEW

File: .github/workflows/Linux_2.12_Node_Tests.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Linux 2.12 App, Chain, Node, and Core Tests
```
Comment (benthecarman) [inline .github/workflows/Linux_2.12_Node_Tests.yml]: should rename

File: .github/workflows/Linux_2.13_Node_Tests.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Linux 2.13 App, Chain, Node, and Core Tests
```
Comment (benthecarman) [inline .github/workflows/Linux_2.13_Node_Tests.yml]: Same here

**Review by benthecarman — CHANGES_REQUESTED** [review]

**Review by benthecarman — APPROVED** [review]

**Review by benthecarman — CHANGES_REQUESTED** [review]
Compile Failing

File: node/src/main/scala/org/bitcoins/node/Node.scala [hunk 3]
```diff
@@ -164,13 +164,12 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
         peerMsgSenders(idx).connect()
         val isInitializedF = for {
           _ <- AsyncUtil.retryUntilSatisfiedF(() => isInitialized(idx),
-                                              maxTries = 1024,
+                                              maxTries = 240,
```
Comment (benthecarman) [inline node/src/main/scala/org/bitcoins/node/Node.scala]: This will make the tor tests much more flaky, maybe bump it up but only when tor is enabled

**Review by Christewart — APPROVED** [review]

**Pull request merged**